src/cow-react/modules/wallet/web3-react/connectors/Injected/index.tsx

```diff
@@ -58,17 +61,24 @@ export class InjectedWallet extends Connector {
         ? desiredChainIdOrChainParameters
         : desiredChainIdOrChainParameters?.chainId
 
-    if (this.connected && desiredChainId && desiredChainId === this.parseChainId(this.provider.chainId)) {
```

Comment:
I would have loved the description of the PR would contain some more information on the changes you are introducing here.

Reply:
I don't think we need this part anymore, they also don't have it in [here](https://github.com/Uniswap/web3-react/blob/de97c00c378b7909dfbd8a06558ed12e1f796caa/packages/metamask/src/index.ts#L130)